### PR TITLE
Jorja test

### PR DESCRIPTION
**Pull request by JorjaF**
## Implements/Fixes: title-here
Completed Model testing for preorders, vendors, and items

## This closes issue:
#26 #27 #28 

## Necessary checkmarks:
- [x] All Tests are Passing
- [x] The code will run locally

## Type of change
- [x] New feature
- [ ] Bug Fix

## Check the correct boxes
- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes
- [ ] No Tests have been changed
- [ ] Some Tests have been changed
- [x] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:
- [ ] My code has no unused/commented out code
- [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

TY!

### CODE REVIEW

File: market_nextdoor_api/tests/models_tests/test_items.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  #     self.item.delete()
+
+  #     self.assertFalse(Item.objects.filter(id=self.item.id).exists())
+
+  #     if item_image_path:
+  #       self.assertFalse(os.path.exists(item_image_path))
+    
+  #   self.assertEqual(Item.objects.count(), 0)
+  #  i can not seem to get this working, i am going to move on and come back to it later
+
```
Comment (EVanGorkom) [inline market_nextdoor_api/tests/models_tests/test_items.py:100]: Are these lines supposed to be commented out?

Comment (JorjaF) [reply market_nextdoor_api/tests/models_tests/test_items.py:100]: For now, I have those commented out on purpose. I worked on it for like 40 minutes, and I figured that I can come back to that.

Comment (EVanGorkom) [reply market_nextdoor_api/tests/models_tests/test_items.py:100]: Gotcha! No worries then. I'll see if I can play with it once it's merged to main and see if I can come up with anything on my end as well!

**Review by EVanGorkom — APPROVED** [review]
Awesome job here! I'll see if I can come up with a solution for the broken test. With this PR we should be over the 80% test coverage!

**Pull request merged**